pkg/compose/run.go

```diff
@@ -127,4 +127,8 @@ func applyRunOptions(project *types.Project, service *types.ServiceConfig, opts
 	for k, v := range opts.Labels {
 		service.Labels = service.Labels.Add(k, v)
 	}
+
+	if opts.ServiceEnvFiles != nil {
+		service.EnvFile = append(service.EnvFile, opts.ServiceEnvFiles...)
```

Comment:
need to make service envfile an absolute path

Reply:
is there any sample code which I can refer in the repository?

Reply:
You can just use:
```go
if !filepath.IsAbs(file) {
					file, err = filepath.Abs(file)
}
```